src/index.js

```diff
@@ -25,30 +25,45 @@ export default declare(({
     SVG_NAME,
     SVG_CODE,
     SVG_DEFAULT_PROPS_CODE,
+    EMIT_DEPRECATED_DEFAULT_PROPS,
   }) => {
+    const defaultProps = SVG_DEFAULT_PROPS_CODE
+      ? 'var props = objectAssign({}, SVG_DEFAULT_PROPS_CODE, overrides);'
```

Comment:
btw, one implication of assign here is that just `props` will never be a stable ref, but I don't think anything in the `SVG_CODE ` will ref just `props` alone so it should be ok...